app/assets/stylesheets/nav.scss

```diff
@@ -36,8 +43,8 @@ body{
 }
 
 body.home {
-  background-size: relative;
-  background-image: url("http://jarques.me/img/potd_sept_30.jpg");
+  background-size: 88em;
+  background-image: url("http://static4.businessinsider.com/image/53a1eb91ecad045e6987bc69-1200/its-hard-to-say-whether-this-is-really-an-office-or-just-a-hip-cafe.jpg");
```

Comment:
URL `http://static4.businessinsider.com/image/53a1eb91ecad045e6987bc69-1200/its-hard-to-say-whether-this-is-really-an-office-or-just-a-hip-cafe.jpg` should not contain protocol or domain